### PR TITLE
added Java resource

### PR DESCRIPTION
**Pull request by lovevictoria2020**
Added a Java resource with tutorials, books, youtube channels and IDEs!

### CODE REVIEW

**Comment by Mayank57** [comment]
everything seems fine except for the 2nd youtube link which should take me to the crash course but the link is for linear algebra course.

File: programming-languages/README.md [hunk 1]
```diff
@@ -56,3 +80,33 @@ It is highly recommended to take the time and get familiar with the standard lib
...
+
+* [Java in Easy Steps](https://www.amazon.com/Java-easy-steps-Covers/dp/1840786213): Easy to follow book that explains the basics of Java with fun and interesting graphics.
+
+* [Head First Java](https://www.amazon.com/dp/0596009208/?tag=javamysqlanta-20): Classic book covering Java that is essental for any beginner programmer.
+
+### Channels
+
+* [TheNewBoston Youtube Channel](https://www.youtube.com/channel/UCJbPGzawDH1njbqV-D5HqKw): This youtube channel is jam packed with tons of resources on learning Java. Are you a beginner? Start with the "Java (Beginner) Tutorial" playlist. When you feel you are ready, move onto to the "Java (Intermediate) Tutorial" playlist to learn some more advanced topics!
+
+* [Derek Banas]()https://www.youtube.com/channel/UCwRXb5dUK4cvsHbx-rGzSgw): Channel with crash course videos teaching you the basic of every programming language!
```
Comment (aditya-shirwatkar) [inline programming-languages/README.md]: Remove the redundant ")" from "()https://.....". It is missing up the rich diff

File: programming-languages/README.md [hunk 2]
```diff
@@ -56,3 +80,33 @@ It is highly recommended to take the time and get familiar with the standard lib
...
+
+### IDEs
+
+* [Eclipse](https://www.eclipse.org/downloads/): One of the most well known IDEs for Java. 
+* [JetBrains IntelliJ IDEA](https://www.jetbrains.com/idea/): A powerful IDE with an extensive library of features. You can get it for free for non-commercial projects as a student!
```
Comment (aditya-shirwatkar) [inline programming-languages/README.md:112]: Replace "basic" with "basics"

Comment (lovevictoria2020) [reply programming-languages/README.md:112]: Fixed it! Check out my latest commit! Thanks so much for bringing that up!

File: programming-languages/README.md (hunk 1 above)
Comment (lovevictoria2020) [reply programming-languages/README.md]: Check out my latest commit for the changes! Thanks so much!

**Comment by lovevictoria2020** [comment]
> everything seems fine except for the 2nd youtube link which should take me to the crash course but the link is for linear algebra course.

Fixed it! It seems that he also had done a linear algebra course as well. Changed it so the link takes you the specific Java crash course! You can check my latest commit to see the edit!

**Review by aditya-shirwatkar — APPROVED** [review]

**Review by take2rohit — APPROVED** [review]

**Comment by take2rohit** [comment]
@lovevictoria2020 Thanks for your contribution. Do star and watch our other repos as well. 👍🏻
Happy Hacking. 😸

**Review by rishesh007 — COMMENTED** [review]
The tutorials of TheNewBoston are very old, also he hadn't updated his lectures. Else all is fine👍.

**Pull request merged**